### DISCUSSION TITLE
Problems pulling Public images

### DISCUSSION THREAD

**Discussion by aitorch** [community/community#26289] [Actions]
<p>when I try to pull a public image (from local or an action) it request for authentication</p>
<pre><code class="lang-auto">docker pull ghcr.io/github/super-linter:latest
latest: Pulling from github/super-linter
df9b9388f04a: Already exists
48763ff2255b: Pulling fs layer
67aa6a9afabb: Pulling fs layer
7fe20896831f: Pulling fs layer
70491ae32830: Waiting
b0c3ebeb5ceb: Waiting
5ceb4a38e435: Waiting
4bed3fe6dd12: Waiting
a3e75de7393b: Waiting
c7e18b86847d: Waiting
8e46ec5613df: Waiting
5da43db05b48: Waiting
620ac8c58403: Waiting
853dc561309d: Waiting
fc993fc2f113: Waiting
0e9afa66b60d: Waiting
33115949fe29: Waiting
f86cf1e783c9: Waiting
a97f57804245: Waiting
19f95280cfde: Waiting
286489327525: Waiting
714cccfb879d: Waiting
af444be4c0d8: Waiting
a1a02c13b3a4: Waiting
fd9c964e0345: Waiting
dfdd9e48ecfa: Waiting
ac43cbd191ba: Waiting
error pulling image configuration: download failed after attempts=1: unauthorized: unauthenticated: User cannot be authenticated with the token provided.
</code></pre>

**Comment by MGough** [comment]
<p>This is happening on other images too: <a href="https://github.com/EnricoMi/publish-unit-test-result-action/issues/307" class="inline-onebox" rel="noopener nofollow ugc">Unauthenticated error when trying to pull image locally or during GitHub Action. · Issue #307 · EnricoMi/publish-unit-test-result-action · GitHub</a></p>
<p>I’ve raised a support ticket as the Status page for github shows no known issues.</p>

**Comment by MGough** [comment, marked as answer]
<p>Resolved before my post could be approved :slightly_smiling_face:</p>
